docs/releases.md

```diff
@@ -1,39 +1,157 @@
-
-## Post-Release
-- [ ] Bump version string in `repo/version.go` to `vX.Y.Z-dev`
-- [ ] Upload the final release to the github releases page: https://github.com/ipfs/go-ipfs/releases
-- Communication
-  - [ ] Create the release issue
-  - [ ] Announcements (both pre-release and post-release)
-    - [ ] Twitter
-    - [ ] IRC
-    - [ ] Reddit
-  - [ ] Blog post (at minimum, paste the changelog. optionally add context and thank contributors.)
-- [ ] Update HTTP-API Documentation on the Website using https://github.com/ipfs/http-api-docs
+- [ ] Automated Testing - Ensure that all tests are passing, this includes:
+  - [ ] unit
+  - [ ] sharness
+  - [ ] [interop](https://github.com/ipfs/interop#test-with-a-non-yet-released-version-of-go-ipfs)
+- [ ] Network Testing:
+  - [ ] test lab things
+- [ ] Infrastructure Testing:
+  - [ ] Deploy new version to a subset of Bootstrappers
+  - [ ] Deploy new version to a subset of Gateways
+  - [ ] Deploy new version to a subset of Preload nodes
+  - [ ] Collect metrics every day. Work with the Infrastructure team to learn of any hiccup
+- [ ] IPFS HTTP Client Libraries Testing:
+  - [ ] [JS](http://github.com/ipfs/js-ipfs-http-client)
+  - [ ] [Go](https://github.com/ipfs/go-ipfs-api)
+- [ ] IPFS Application Testing -  Run the tests of the following applications:
+  - [ ] WebUI
+  - [ ] IPFS Desktop
+  - [ ] IPFS Companion
+  - [ ] NPM on IPFS
+
+### Release Stage 2 - Invite IPFS partners to test
+
+- [ ] Reach out to the following IPFS partners for testing this release (check when no more problems have been reported):
+  - [ ] Infura
+  - [ ] Textila
+  - [ ] Open Bazaar
+  - [ ] QRI
+- [ ] Run tests available in the following repos with the latest RC (check when all tests pass):
+  - [ ] [orbit-db](https://github.com/orbitdb/orbit-db)
+
+PSA: If you are a heavy user of `go-ipfs`, have developed a solid test infrastructure for your application and would love to help us would like to help us test `go-ipfs` release candidates, reach out to go-ipfs-wg@ipfs.io.
+
+### Release Stage 3 - Announce to the broader community
+
+- [ ] Documentation
+  - [ ] Ensure that CHANGELOG.md is up to date
+  - [ ] Ensure that README.md is up to date
+  - [ ] Ensure that all the examples we have produced for go-ipfs run without problems
+  - [ ] Update HTTP-API Documentation on the Website using https://github.com/ipfs/http-api-docs
+- [ ] Invite the community through (link to the release issue):
+  - [ ] [discuss.ipfs.io](https://discuss.ipfs.io/c/announcements)
+  - [ ] Twitter
+
+### Release Stage 4 - Complete the Release
+
+- Take a snapshot between of everyone that has contributed to this release (including its subdeps in IPFS, libp2p, IPLD and multiformats) using [`name-your-contributors`](https://www.npmjs.com/package/name-your-contributors). Generate a nice markdown list with [this script](https://gist.github.com/alanshaw/5a2d9465c5a05b201d949551bdb1fcc3). Add it to this issue.
```

Comment:
This gist needs to be updated to point to the go code repos. Thanks to @alanshaw for creating it in the first place.

Reply:
The `bin/mkreleaselog` script currently does this. Output: https://blog.ipfs.io/93-go-ipfs-0.4.21#huge-thank-you-to-everyone-that-made-this-release-possible